docs/specification/draft/basic/enterprise.mdx

```diff
@@ -0,0 +1,294 @@
+| `resource`             | REQUIRED          | The Issuer URL of the MCP server's authorization server.                                                                                                                       | `https://mcp.example.com/auth`                                         |
+| `audience`             | MUST NOT be used  | The audience parameter MUST NOT be used to indicate the authorization server.                                                                                                  | —                                                                      |
+| `scope`                | OPTIONAL          | The space-separated list of scopes at the MCP Server that are being requested.                                                                                                 | `scope1 scope2`                                                        |
+| `subject_token`        | REQUIRED          | The identity assertion (e.g. the OpenID Connect ID Token or SAML assertion) for the target end-user.                                                                           | (JWT or SAML assertion string)                                         |
+| `subject_token_type`   | REQUIRED          | Indicates the type of the security token in the `subject_token` parameter, as specified in [RFC8693 Section 3](https://datatracker.ietf.org/doc/html/rfc8693#section-3).      | `urn:ietf:params:oauth:token-type:id_token` (OIDC)<br/>`urn:ietf:params:oauth:token-type:saml2` (SAML) |
+
+Additional parameters defined in [Section 2.1 of RFC8693](https://datatracker.ietf.org/doc/html/rfc8693#name-request) (`actor_token` and `actor_token_type`) are not used in this specification.
+
+If the IdP requires client authentication when the MCP client performs OpenID Connect for single sign-on, then client authentication of the Token Exchange request is also required.
+
+The example below illustrates the Token Exchange request, using an ID Token as the Identity Assertion and a client secret as the client authentication method. The ID token is passed as a Subject Token.
+
+```
+POST /oauth2/token HTTP/1.1
+Host: acme.idp.example
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=urn:ietf:params:oauth:grant-type:token-exchange
+&requested_token_type=urn:ietf:params:oauth:token-type:id-jag
+&resource=https://acme.chat.example/
+&scope=chat.read+chat.history
+&subject_token=eyJraWQiOiJzMTZ0cVNtODhwREo4VGZCXzdrSEtQ...
+&subject_token_type=urn:ietf:params:oauth:token-type:id_token
+&client_id=2ec954a1d60620116d36d9ceb7
+&client_secret=a26d84873504215a34a86d52ef5cd64f4b76
+```
+
+### 4.1 Processing Rules
+
+The IdP MUST validate the subject token, and MUST validate that the audience of the Subject Token (e.g. the `aud` claim of the ID Token) matches the `client_id` of the MCP client's client authentication.
```

Comment:
What if you split the example out of the explanation? Is this clearer?

```suggestion
The IdP **MUST** validate the Subject Token, and **MUST** validate that the audience of the Subject Token matches the `client_id` of the MCP Client. For example, the IdP could check that the `aud` claim of the ID Token (audience of the Subject Token) matches the `client_id` parameter in the token request (OAuth client authentication for the token request).
```

Agree that we want to make sure this is implemented correctly.